roles/installer/tasks/migrate_schema.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Check for pending migrations
```

Comment:
:+1: 
This task ensures that the k8s job is only run if there are new schema changes to be migrated.  

And the unique name based on the version and the random hash ensure that there are no conflicts with other k8s jobs from previous migrations.